css/src/components/form/radio.scss

```diff
@@ -42,30 +40,26 @@ $radio-spacing: 0.5em !default;
 		height: $radio-circle-size;
 		border: $radio-border-width solid $radio-border-color;
 		border-radius: $border-radius-rounded;
+		background-color: $body-background;
 		color: $radio-dot-color;
 		appearance: none;
+		content: '';
```

Comment:
nit: can you add here `cursor: pointer;`? For some reason, I get the default arrow, would be nice to see the pointer :)

Reply:
Added!